package Exceptii;

import static Exceptii.TestThrow1.validate;

public class ThrowMain {
    public static void main(String[] args) {
        validate(21);
        System.out.println("Restul codului...");
    }
}
